src/player.cpp

```diff
@@ -12703,3 +12703,63 @@ void player::do_skill_rust()
         }
     }
 }
+
+std::pair<std::string, nc_color> player::get_hunger_description() const
```

Comment:
I think this needed to be player::hunger_stat()

Reply:
no but thanks for point at this, i wouldn't have noticed that hunger_stat was a duplicate until after i was doing more with the UI